scale/docs/rest/v6/strike.rst

```diff
@@ -0,0 +1,496 @@
++--------------------+-------------------+--------------------------------------------------------------------------------+
+| .job               | JSON Object       | The job that is associated with the Strike process.                            |
+|                    |                   | (See :ref:`Job Details <rest_job_details>`)                                    |
++--------------------+-------------------+--------------------------------------------------------------------------------+
+| .created           | ISO-8601 Datetime | When the associated database model was initially created.                      |
++--------------------+-------------------+--------------------------------------------------------------------------------+
+| .last_modified     | ISO-8601 Datetime | When the associated database model was last saved.                             |
++--------------------+-------------------+--------------------------------------------------------------------------------+
+
+.. _rest_v6_strike_create:
+
+**Example POST /v6/strikes/ API call**
+
+Request: POST http://.../v6/strikes/
+
+ .. code-block:: javascript 
+ 
+    { 
+        "title": "My Strike Process", 
+        "description": "This is my Strike process for detecting my favorite files!", 
+        "configuration": { 
+            "version": "2.0", 
+            "workspace": "my-workspace", 
+            "monitor": { 
+                "type": "dir-watcher", 
+                "transfer_suffix": "_tmp" 
+            }, 
+            "files_to_ingest": [{ 
+                "filename_regex": ".*txt" 
+            }] 
+        } 
+    } 
+
+Response: 201 Created
+Headers:
+Location http://.../v6/strikes/105/
+
+ .. code-block:: javascript 
+ 
+    { 
+        "id": 1, 
+        "name": "my-strike-process", 
+        "title": "My Strike Process", 
+        "description": "This is my Strike process for detecting my favorite files!", 
+        "job": { 
+            "id": 7, 
+            "job_type": { 
+                "id": 2, 
+                "name": "scale-strike", 
+                "version": "1.0", 
+                "title": "Scale Strike", 
+                "description": "Monitors a directory for incoming source files to ingest", 
+                "category": "system", 
+                "author_name": null, 
+                "author_url": null, 
+                "is_system": true, 
+                "is_long_running": true, 
+                "is_active": true, 
+                "is_operational": true, 
+                "is_paused": false, 
+                "icon_code": "f0e7" 
+            }, 
+            "job_type_rev": { 
+                "id": 2 
+            }, 
+            "event": { 
+                "id": 1 
+            }, 
+            "status": "RUNNING", 
+            "priority": 10, 
+            "num_exes": 1 
+        }, 
+        "configuration": { 
```

Comment:
We need a v6 Strike configuration JSON. For right now, just create a v6 JSON class (no changes from v2) and we'll worry about a separate logic class later. Do the version stripping (like in batch APIs) so the version '2.0' or '6' doesn't appear anywhere in the responses.